TYPO3.Flow/Classes/TYPO3/Flow/Core/Booting/Scripts.php

```diff
@@ -490,18 +490,6 @@ protected static function compileDoctrineProxies(Bootstrap $bootstrap)
     }
 
     /**
-     * Initializes the persistence framework
-     *
-     * @param Bootstrap $bootstrap
-     * @return void
-     */
-    public static function initializePersistence(Bootstrap $bootstrap)
-    {
-        $persistenceManager = $bootstrap->getObjectManager()->get(\TYPO3\Flow\Persistence\PersistenceManagerInterface::class);
-        $persistenceManager->initialize();
```

Comment:
This is no longer called with this change, so the call needs to be called elsewhere. `initializeObject()` might be enough, even…